src/app/components/pagination.tsx

```diff
@@ -13,8 +13,11 @@ export default function Pagination({
     return null;
   }
   return (
-    <nav className={styles.nav}>
-      <ul className={styles.ul}>
+    <nav className={styles.nav} aria-labelledby="table-pagination">
+      <h3 id="table-pagination" className="visually-hidden">
+        Table pagination
```

Comment:
```suggestion
        Пагінація
```
Я збираюсь цей компонент ще для пагінації в пошуку використати